runtime-modules/governance/src/council.rs

```diff
@@ -75,6 +85,73 @@ impl<T: Trait> Module<T> {
         CouncilMint::<T>::put(mint_id);
         Ok(mint_id)
     }
+
+    fn add_reward_relationship(
+        destination: &T::AccountId,
+    ) -> Result<T::RewardRelationshipId, recurringrewards::RewardsError> {
+        if let Some(reward_source) = Self::council_mint() {
+            let recipient = <recurringrewards::Module<T>>::add_recipient();
+
+            let next_payout_at = system::Module::<T>::block_number()
+                + Self::first_payout_after_term_start()
+                + T::BlockNumber::one();
+
+            let relationship_id = <recurringrewards::Module<T>>::add_reward_relationship(
+                reward_source,
+                recipient,
+                destination.clone(),
+                Self::amount_per_payout(),
+                next_payout_at,
+                Self::payout_interval(),
+            )?;
+
+            RewardRelationships::<T>::insert(destination, relationship_id);
+
+            Ok(relationship_id)
+        } else {
+            Err(recurringrewards::RewardsError::RewardSourceNotFound)
+        }
+    }
+
+    fn remove_reward_relationships() {
+        for seat in Self::active_council().into_iter() {
+            if RewardRelationships::<T>::exists(&seat.member) {
```

Comment:
Why is this `exists` test here? This implies its possible for the council member to have no reward?

Reply:
Yes. There are two reasons for such a case.

1. After a runtime upgrade the existing council doesn't have rewards (I didn't add any migration step or default values for rewards)

2. There are helper methods for adding and setting a council directly with a dispatchable,  insert_member() and set_council().  I think we should get rid of these methods and only allow council to be elected, so I didn't bother to add rewards for seats added with these methods.

Reply:
From internal discussion: the first council post migration has no reward relationships, so this must deal with this.

Reply:
So we ignore this issue in this release, made an issue for the future here
https://github.com/Joystream/joystream/issues/307